src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
I lowered the accuracy but the result is still different (because of rounding errors, convergence procedure, ...) which is worth pointing at. Should I add a line that explains it ?
Alos, I find it amusing that your interpretation is that OpenMDAO lacks accuracy compared to a dichotomy and not the other way around (:

Reply:
<p>I see now 1065.8kg for both computations. Don't you ?</p><p> </p><p>And about accuracy, I would rather say that I know how easy it is to tweak solver tolerances in OpenMDAO, and also that it was the easiest setting I could check. But my second move would have been to check the tolerance for fsolve. BTW, if you fear that results could not match, I suggest to lower both tolerances (OpenMDAO <strong>and </strong>fsolve).</p><p> </p><p> </p>

Reply:
For the MDA, the results of the last 3 notebooks match but the results of the first one is slightly different, I raised the accuracy (0.1 to 0.01) so now all results match.

Reply:
Also checked that all MDO gives the same results when rounding. Both done in bdb94be2